util/masking.py

```diff
@@ -35,6 +37,12 @@
             "accepted by the filtering algorithm."
     .type = float(value_min=0)
 
+  parallax_correction = None
```

Comment:
Why `None` not `True`?

Reply:
I went with this name for consistency with https://github.com/dials/dxtbx/blob/2caec59df9e63219892e152d47dbdef2afc845f4/model/detector.py#L108-L112. I didn't want to set the default to `True` in case that implied that it would enable `parallax_correction` even if the detector model didn't already specify it. Only setting it to `False` will have any effect. Perhaps `disable_parallax_correction=False` would be clearer in this context? Welcome suggestions here...

Reply:
Without wishing to bikeshed, I agree that `disable_parallax_correction=False` is more honest (had occurred to me as well)